newjersey/navigator.business.nj.gov
# feat: [#187871332] update outage bar to be sticky; refactor logo only…

## Description


- Makes the outage bar sticky by moving it inside of the Mobile + Desktop navs
- Refactors LogoOnly nav bars for mobile and desktop to use their respective wrappers
- Adds alert role to outage bar

### Ticket



This pull request resolves [#187871332](https://www.pivotaltracker.com/story/show/187871332).

### Approach



### Steps to Test


- Force the outage bar to always render
- Ensure it's sticky as you scroll in all screen sizes

### Notes



## Code author checklist
